### PR TITLE
Exclude Python conformance utils from Azure pre-commit

### PR DESCRIPTION
**Pull request by iefode**
### Details:
 - *Exclude Conformance python tools from Azure pre-commit*

### Tickets:
 - *93965*

### CODE REVIEW

File: .ci/azure/android_arm64.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ trigger:
     - /**/docs/*
     - /**/*.md
     - README.md
+    - src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/*
```
Comment (ilyachur) [inline .ci/azure/android_arm64.yml:12]: Should it be
```suggestion
    - src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/**
```
?
Did you check that these changes work?

Comment (iefode) [reply .ci/azure/android_arm64.yml]: fixed

Comment (ilyachur) [reply .ci/azure/android_arm64.yml]: Did you check that these paths works as expected? :)

File: .ci/azure/linux_cuda.yml [hunk 2]
```diff
@@ -21,6 +23,8 @@ pr:
     - /**/docs/*
     - /**/*.md
     - README.md
+    - src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/**
+    - src/tests/functional/plugin/conformance/**
```
[the remark this answers is not in the document]
Comment (iefode) [reply .ci/azure/linux_cuda.yml:27]: Not for all (we should check build)

File: .ci/azure/android_arm64.yml (hunk 1 above)
Comment (iefode) [reply .ci/azure/android_arm64.yml]: I made sure :)

File: .ci/azure/linux_arm64.yml [hunk 3]
```diff
@@ -8,7 +8,8 @@ trigger:
     - docs/
     - /**/docs/*
     - /**/*.md
-    - README.md
```
Comment (ilyachur) [inline .ci/azure/linux_arm64.yml:11]: Why do you remove README.md?

**Review by ilyachur — APPROVED** [review]

**Pull request merged**